docs/include/slack.txt

```diff
@@ -1,3 +1,5 @@
 Welcome to the Write the Docs Slack! We're glad you're here, and we look forward to your contributions to our community.
 
 We've written some guidelines to help you get the most out of this Slack. Please take a few minutes to learn about some of our channels, and familiarize yourself with our social rules. All this information is at https://www.writethedocs.org/slack/ -- make sure to look through the whole page. We are grounded in the principles of open source, and we pride ourselves on being one of the most welcoming communities in tech. Our social rules are an important tool to further this goal!
+
+If you're new to Write the Docs, we also invite you to learn more about our community. Check out how we got started at https://www.writethedocs.org/origin-story/, read who's on the current team at https://www.writethedocs.org/team/, and learn about who we are at https://www.writethedocs.org/about-us/. There's a lot more on the website, too -- explore, get involved, and welcome!
```

Comment:
Looks great! 

Unless we can add link syntax I'd be tempted to break this up for readability

* awesome fun text - link
* more fun  - more link

Reply:
Yeah, lazy me had the same thought last night. Fixing up ...